### PR TITLE
Ensure latest version of curl is installed

### PR DESCRIPTION
**Pull request by tjoelsson**
We need latest curl to make outbound https connections.

@juwai/platform-all please review

### CODE REVIEW

File: tasks/system_packages.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@
     - mysql55
     - mysql55-devel
   when: ansible_distribution == 'Amazon'
+
+- name: ensure latest version of curl
```
Comment (stuartwan) [inline tasks/system_packages.yml]: ? ensure latest version of curl... and?

Comment (tjoelsson) [inline tasks/system_packages.yml]: seriously?

**Comment by tjoelsson** [comment]
@imlazyone better now?

File: tasks/system_packages.yml (hunk 1 above)
Comment (stuartwan) [inline tasks/system_packages.yml]: well just saw all above says "ensure {package} present"

**Comment by agirivera** [comment]
:+1: Looks good

**Comment by stuartwan** [comment]
👍

**Comment by tjoelsson** [comment]
@imlazyone please merge

**Pull request merged**